Cargo.toml

```diff
@@ -20,12 +20,13 @@ metal = ["wgn/gfx-backend-metal"]
 dx11 = ["wgn/gfx-backend-dx11"]
 dx12 = ["wgn/gfx-backend-dx12"]
 vulkan = ["wgn/gfx-backend-vulkan"]
-gl = ["wgn/gfx-backend-gl"]
+#gl = ["wgn/gfx-backend-gl"]
```

Comment:
Is there an issue for the part that gl is blocked on here? It would probably be useful to clarify it in a comment for anyone using gl patches at the moment

Reply:
issue is that we initialize all other backends without `winit` feature
for GL, there is currently no way to use it without `glutin`. Enabling `glutin` though on the backend would conflict with `glutin` in wgpu-rs itself, since the backends are from crates.io and they rely on the old `glutin` version...
This will be unlocked if/when we back-port raw-window-handle to gfx-backend-gl-0.3